pom.xml

```diff
@@ -158,10 +158,9 @@
         <ddf-spatial>DDF Spatial</ddf-spatial>
         <ddf-ui>DDF Search UI</ddf-ui>
 
-        <asciidoctor.maven.plugin.version>1.5.2</asciidoctor.maven.plugin.version>
-        <asciidoctorj-pdf.version>1.5.0-alpha.11</asciidoctorj-pdf.version>
-        <asciidoctorj.version>1.5.3.2</asciidoctorj.version>
-        <jruby.version>9.1.2.0</jruby.version>
+        <asciidoctor.maven.plugin.version>1.5.3</asciidoctor.maven.plugin.version>
+        <asciidoctorj.pdf.version>1.5.0-alpha.11</asciidoctorj.pdf.version>
+        <jruby.version>1.7.25</jruby.version>
```

Comment:
JRuby's releases are weird, it goes from 1.7.26 straight to 9.1.6.0. Asciidoctor uses 1.7.25 though.